kairos-server/Cargo.toml

```diff
@@ -16,6 +16,7 @@ default = ["cctl-tests", "deposit-mock"]
 all-tests = ["cctl-tests", "deposit-mock"]
 cctl-tests = []
 deposit-mock = []
+database = ["kairos-data"]
```

Comment:
The feature should probably be added to `all-tests`

Reply:
Sure